Sources/IBLinterKit/Rules/AmbiguousViewRule.swift

```diff
@@ -31,7 +31,7 @@ extension Rules {
         private func validate<T: InterfaceBuilderFile>(for view: ViewProtocol, file: T) -> [Violation] {
             let violation: [Violation] = {
                 if view.isAmbiguous ?? false {
-                    let message = "\(view.customClass ?? view.elementClass) (\(view.id)) has ambiguous constraints"
+                    let message = "\(view.customClass ?? view.elementClass) has ambiguous constraints"
```

Comment:
`id` property is removed on this commit. https://github.com/IBDecodable/IBDecodable/commit/4b7f278c544e5b2de6ebfbfd411988e01dbeac6a#diff-3bee56c74770556bf81f46fc6772b3e2L14

So I dropped these outputs.
I think they can be optional...

Reply:
I think we can get `id` by casting `IBIdentifiable` and it's better than making all `id` as optional. 🤔

Reply:
I added `viewName` util. How about this?

https://github.com/IBDecodable/IBLinter/pull/97/commits/00de06364ed935c3c949ae1a625ffa2ac8461da1

Reply:
Looks good 👍 👍